x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/components/pipeline_processors_editor_item/pipeline_processors_editor_item.scss

```
@@ -65,4 +64,20 @@
     // Prevent content jump when spinner renders
     min-width: 15px;
   }
+
+  &__controlsContainer {
+    // Make sure this element knows how wide it is
+    width: 100%;
+    // The last element in these controls is an editable text description that can contain an unknown amount (i.e., width) of text.
+    overflow: hidden;
+  }
+
+  // By default, flex sets the element width to "auto", we set this explicitly again to avoid the flex item growing beyond the width given
+  // by flex. This applies to both of the classes below
+  &__controlsFlexItem {
```

You could combine these into a single rule since they share the same, single style.

```suggestion
  &__controlsFlexItem, &__descriptionContainer {
    min-width: 0;
  }
  ```
```